accumulators/capacity/capacity.go

```diff
@@ -1,239 +0,0 @@
-// Copyright 2020 New Relic Corporation. All rights reserved.
```

Comment:
There was a task to remove capacity in the backlog. My only doubt is if someone has some alerts on this